sample-code/src/index.ts

```diff
@@ -0,0 +1,39 @@
+import express from 'express';
+import { chatCompletion, computeEmbedding } from './aiservice.js';
+
+const app = express();
+const port = 8080;
+
+app.get(['/', '/health'], (req, res) => {
+  res.send('Hello World! 🌍');
+});
+
+app.get('/llm', (req, res) => {
+  chatCompletion()
+    .then((response) => {
+      res.send(response);
+    })
+    .catch((error) => {
+      console.error(error);
+      res.status(500).send('Yikes, vibes are off apparently 😬 -> ' + error.message);
+    })
```

Comment:
[pp] Consider switching this (and the other code below) to the more modern await
```suggestion
  try {
    res.send(await chatCompletion());
  } catch (error) {
    console.error(error);
    res.status(500).send('Yikes, vibes are off apparently 😬 -> ' + error.message);
  }
```